DRODLib/CurrentGame.cpp

```diff
@@ -6770,6 +6781,8 @@ void CCurrentGame::SetMembersAfterRoomLoad(
 	this->bHalfTurn = false;
 	this->bWaitedOnHotFloorLastTurn = false;
 	this->conquerTokenTurn = NO_CONQUER_TOKEN_TURN;
+	this->bWasRoomConqueredAtTurnStart = false;
```

Comment:
Suggest we also init these new vars in Clear(), just in case.

Reply:
Good point! Also was not resetting `bIsLeavingLevel` here.